DependencyInjection/CmfRoutingExtension.php

```diff
@@ -240,13 +240,15 @@ public function loadPhpcrProvider($config, XmlFileLoader $loader, ContainerBuild
 
         $container->setParameter(
             $this->getAlias() . '.dynamic.persistence.phpcr.route_basepaths',
-            $config['route_basepaths']
+            array_values(array_unique($config['route_basepaths']))
```

Comment:
array_values call is needed because the array keys need to match as well, in order to let the tests pass.